miden-lib/asm/kernels/transaction/main.masm

```diff
@@ -64,13 +64,25 @@ proc.main.1
     # Prologue
     # ---------------------------------------------------------------------------------------------
 
+    # add redundant instructions to make span non-empty
+    push.0 drop
+    trace.131072
+
     # execute the transaction prologue
     exec.prologue::prepare_transaction
     # => []
 
+    # add redundant instructions to make span non-empty
+    push.0 drop
+    trace.131073
+
     # Note Processing
     # ---------------------------------------------------------------------------------------------
 
+    # add redundant instructions to make span non-empty
+    push.0 drop
+    trace.131074
```

Comment:
In addition to this, we should also capture execution of cycle count of each note - so, we need more tracing in the while-loop below.